sql_csv_utils.py

```diff
@@ -288,10 +291,30 @@ def create_update_statement(self, tab_name: str, col_list: list, val_list: list,
         return sql
 
     def taxon_get(self, name, hybrid=False, taxname=None):
+        """taxon_get: function to retrieve taxon id from specify database:
+            args:
+                name: the full taxon name to check
+                hybrid: whether the taxon name belongs to a hybrid
+                taxname: the name ending substring of a taxon name, only useful for retrieving hybrids.
+        """
 
```

Comment:
Context - I presume you've done a "lower" at some point, but you might want to double check.